<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Difficulty</class>
 <widget class="QMainWindow" name="Difficulty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources.qrc">
    <normaloff>:/resources/icon.ico</normaloff>:/resources/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 253, 230);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Freestyle Script</family>
        <pointsize>28</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Select Difficulty</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="easyButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MD아트체</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{border: none; background-color: rgb(255, 205, 247);}
QToolButton:hover{background-color: rgb(243, 175, 255);}</string>
        </property>
        <property name="text">
         <string>Easy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>HY동녘M</family>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>For immature conductor
Expert Orchestra
They can do well themselves</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="normalButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MD아트체</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{border: none; background-color: rgb(255, 205, 247);}
QToolButton:hover{background-color: rgb(243, 175, 255);}</string>
        </property>
        <property name="text">
         <string>Normal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>HY동녘M</family>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>For intermediate conductor
Amateur Orchestra
They need your direction</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QToolButton" name="hardButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MD아트체</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{border: none; background-color: rgb(255, 205, 247);}
QToolButton:hover{background-color: rgb(243, 175, 255);}</string>
        </property>
        <property name="text">
         <string>Hard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>HY동녘M</family>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>For expert conductor
Improvised Orchestra
They can't do anything without you</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QToolButton" name="backButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MD아트체</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{border: none; background-color: rgb(255, 205, 247);}
QToolButton:hover{background-color: rgb(243, 175, 255);}</string>
      </property>
      <property name="text">
       <string>Back to menu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
